Assets/Reflex/Editor/DebuggingWindow/EditorIcons.cs

```diff
@@ -92,13 +93,13 @@ void SaveIcon( string icon_name )
                     }
                     catch (System.Exception e)
                     {
-                        Debug.LogError("Cannot save the icon : " + e.Message);
+                        DebugLogger.LogError("Cannot save the icon : " + e.Message);
```

Comment:
This 3rd party class can be entirely removed, I used it for finding the icons but forgot to remove it.

Reply:
Done